packages/manager/modules/pci/src/projects/project/notebooks/dashboard/dashboard.routing.js

```diff
@@ -1,4 +1,6 @@
 import find from 'lodash/find';
+import Notebook from '../Notebook.class';
+import { NOTEBOOK_POLLER_NAMESPACES } from '../Notebook.constants';
```

Comment:
Since we follow `camecase` right now for `cosntants` file, i think its better to keep it as `notebook.constants.` to keep it uniform across the manager.

Reply:
Done